explainaboard/utils/py_utils_test.py

```diff
@@ -0,0 +1,15 @@
+"""Tests for py_utils.py."""
+
+from __future__ import annotations
+
+import math
+import unittest
+
+from explainaboard.utils.py_utils import replace_nan
+
+
+class ReplaceNanTest(unittest.TestCase):
```

Comment:
```suggestion
class ReplaceNanTest(unittest.TestCase) -> None:
```

Reply:
it's a little strange to put `-> None` at the end of the class, which will result in an error. So I put them at the end of the test function.

Reply:
Sorry this is my bad. Please ignore this comment.